packages/cardano-services/src/cli.ts

```diff
@@ -67,7 +67,9 @@ import { EPOCH_POLL_INTERVAL_DEFAULT } from './util';
 import { HttpServer } from './Http';
 import { PgBossQueue, isValidQueue } from './PgBoss';
 import { ProjectionName } from './Projection';
+import { URL } from 'url';
```

Comment:
As far as I remember the SDK often (always?) uses the built-in `URL` type and in some PRs I had to remove the imported `URL` type as it was causing some problems (sorry I can't remember about details).

I would remove this line; everything should work as it does with this line.

Reply:
Agree, it is available as global in all target environments https://developer.mozilla.org/en-US/docs/Web/API/URL

Reply:
I had no problems importing it. Removed here 9b2db5b